src/contract/contract.ts

```diff
@@ -1,11 +1,9 @@
-import { Contract as EthersContract } from 'ethers'
-
 import { Account } from '../account'
 import type { Address, ContractFactory } from '../types'
 import { isAddress } from '../types'
 
 /** ### Contract Instance */
-export class Contract<C extends EthersContract> {
+export class Contract<C extends { address: Address }> {
```

Comment:
it fails to build on my local machine without build cache

Reply:
try `pnpm build`

Reply:
build could be fixed via

`return (this.contract as unknown as { address: Address }).address` in address getter